File: HacktoberFestContribute/Algorithms/Power of Two/Golang/power_of_two.go
package main

import (
	"fmt"
)

func main() {
	for i := 0; i < 1025; i++ {
		if ((i > 0) && (i & (i - 1)) == 0) {
			fmt.Println(i, "is power of two")
		}
	}
}

File: HacktoberFestContribute/Algorithms/Power of Two/power_of_two.cpp
#include <cstdio>

int main()
{
 int number,c;
 printf("Enter a number to check if it's a power of 2!\n");
 scanf("%d",&number);
 c=number;
 while(number>2 && number%2==0)
       {
        number/=2;
       }
 printf("%d %s a power of 2\n",c,number==2?"it's":"it's not");
 return 0;
}
